src/components/elements/MenuLink.js

```diff
@@ -12,7 +13,13 @@ const MenuLink = (type, id, onClick, dispatch) => {
       key={`${type}-${id}` }
       primaryText={<Link to={typeRoute} >{`${typeTitle} ${id}`}</Link>}
       rightIcon={<ActionDelete
-        onClick={() => dispatch(onClick(id)) } />}
+        onClick={
+          () => {
+            dispatch(onClick(id))
+            persistor.purge()
```

Comment:
I'm not sure the MenuLink component is the best place for this logic. Maybe move to the reducer or action?